### PR TITLE
blogging.rst: Minor edits

### PR DESCRIPTION
**Pull request by RJ722**
Keep the line length below 80, make the language more explicit.

### CODE REVIEW

**Comment by RJ722** [comment]
@roshnet can you review this once, please?

File: docs/source/blogging.rst [hunk 1]
```diff
@@ -19,23 +21,27 @@ Below are the steps you need to perform in order to get started.
...
 Configure your blog space
 -------------------------
 
 Make your blog visible to everyone
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-By default, all the posts you write are stored as drafts in your blog.
-To make them public, follow these steps -
- - navigate to `your blog <https://blogs.amuoss.tk/me/c/>`_
- - open the `customize <https://blogs.amuoss.tk/me/c/roshan>`_ link. This opens up your blog settings page.
```
Comment (roshnet) [inline docs/source/blogging.rst:37]: Oops, my bad. How could I forget to remove my username!

File: docs/source/blogging.rst [hunk 2]
```diff
@@ -19,23 +21,27 @@ Below are the steps you need to perform in order to get started.
...
- - Under the *Publicity* section, change *Private* to **Public**.
+By default, all the posts you write are unlisted (i.e. they do not appear on
+the https://blogs.amuoss.tk/read).
```
Comment (roshnet) [inline docs/source/blogging.rst]:
```suggestion
the `reader <https://blogs.amuoss.tk/read>`).
```

**Review by roshnet — CHANGES_REQUESTED** [review]

File: docs/source/blogging.rst (hunk 2 above)
Comment (roshnet) [reply docs/source/blogging.rst]: @RJ722

Comment (RJ722) [reply docs/source/blogging.rst]: This was intentional because because when we mention it for the first time, they won't know what a reader is.

Comment (RJ722) [reply docs/source/blogging.rst]: I need to remove the preceding "the" though.

**Review by roshnet — APPROVED** [review]
@RJ722 changes look good!

**Pull request merged**